infra/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "digitalocean" {
+  # You need to set this in your .bashrc
+  # export DIGITALOCEAN_TOKEN="Your API TOKEN"
+  #
+}
+
+resource "digitalocean_droplet" "delta_droplet" {
+  ssh_keys           = [8187378]
```

Comment:
Without knowing anything about tf, this seems like something that maybe shouldn't be here?